### PR TITLE
Fix vtx export

### PR DESCRIPTION
**Pull request by Ozaq**
jpscore was writing goals.vtk file even though writing vtk files was
disabled.

Fixes: #476

### CODE REVIEW

File: routing/ff_router/ffRouter.cpp [hunk 1]
```diff
@@ -106,7 +106,9 @@ bool FFRouter::Init(Building* building)
           _goalToLineUIDmap = _globalFF->getGoalToLineUIDmap();
           _goalToLineUIDmap2 = _globalFF->getGoalToLineUIDmap2();
           _goalToLineUIDmap3 = _globalFF->getGoalToLineUIDmap3();
-          _globalFF->writeGoalFF("goal.vtk", goalIDs);
+          if(_building->GetConfig()->get_write_VTK_files()) {
+              _globalFF->writeGoalFF("goal.vtk", goalIDs);
```
Comment (gjaeger) [inline routing/ff_router/ffRouter.cpp]: The element <write_VTK_files>false</write_VTK_files> works as desired.

Unfortunately the input true does not generate any output (goal.vtk).

Comment (Ozaq) [reply routing/ff_router/ffRouter.cpp]: Hmm strange... Ill look into it

Comment (Ozaq) [reply routing/ff_router/ffRouter.cpp]: Ok it looks like `write_VTK_files` has to be an element under FFRouter params, e.g.:
```xml
<route_choice_models>
  <router router_id="1" description="ff_global_shortest">
    <parameters>
      <write_VTK_files>true</write_VTK_files>
    </parameters>
  </router>
</route_choice_models>
```

Additionally the same XML element is reused under 
```xml
<operational_models>
  <model operational_model_id="3" description="Tordeux2015">
    <model_parameters>
      <write_VTK_files>true</write_VTK_files>
    </model_parameters>
```
But here this sets `write_vtk_direction` on config.

@chraibi Can you help me out here? Is this the desired behavior or is there more fix?

Comment (zeroset) [reply routing/ff_router/ffRouter.cpp]: `write_VTK_files` is a Parameter of the FFRouters and therefore should be part of the parameters of the router. 
Additionally I created a new issue for renaming the model specific `write_VTK_files` parameter (#484).

**Pull request merged**